dapp/src/components/dashboard/pages/Users/index.jsx

```diff
@@ -52,7 +56,7 @@ const Users = ({
   const { communityAddress, isClosed } = community
   const [data, setData] = useState([])
   const [search, setSearch] = useState('')
-  const apiRoot = getApiRoot(network.networkType === 'fuse' ? 'default' : network.networkType)
+  const apiRoot = getApiRoot(useSelector(getForeignNetwork))
   let url = `${apiRoot}/entities/${communityAddress}?type=user`
```

Comment:
why not use use `fetchBusinessesEntities`? I don't like the use of `getApiRoot` and the `url` here.. It's better to keep that stuff from the components if it's not a special reason for it